release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -624,6 +624,15 @@ For more information, see xref:../scalability_and_performance/node-observability
 [id="ocp-4-11-auth"]
 === Authentication and authorization
 
+[id="ocp-4-11-post-installation"]
+=== Post-installation configuration
```

Comment:
Just tagging @opayne1 to make sure there are no objections to adding a new high-level section for new features!

Or maybe she has some suggestion for its placement? I'm wondering whether it would make sense to add it right after the install section?

Reply:
I think it would be okay to add another section. We could always remove that section in a future release if there is no content to add to it. 

+1 to moving it after the install section. I think that is where most users would expect to see it.

Reply:
fixed.